vpr/src/base/vpr_constraints_writer.cpp

```diff
@@ -138,7 +142,7 @@ void setup_vpr_floorplan_constraints_cutpoints(VprConstraints& constraints, int
             int ymax = vertical_cuts[j + 1] - 1;
 
             Region reg;
-            reg.set_region_rect(xmin, ymin, xmax, ymax);
+            reg.set_region_rect({xmin, ymin, xmax, ymax, 0});
```

Comment:
Why always layer 0?

Reply:
Could assert that it is single die, or could loop to write out constraints for each layer.